cmd/virt-launcher/virt-launcher.go

```diff
@@ -110,16 +110,16 @@ func startCmdServer(socketPath string,
 }
 
 func createLibvirtConnection(runWithNonRoot bool) virtcli.Connection {
-	libvirtUri := "qemu:///system?socket=/run/libvirt/libvirt-sock"
+	libvirtUri := "qemu:///system?socket=/run/libvirt/virtqemud-sock"
 	user := ""
 	if runWithNonRoot {
 		user = putil.NonRootUserString
-		libvirtUri = "qemu+unix:///session?socket=/var/run/libvirt/libvirt-sock"
+		libvirtUri = "qemu+unix:///session?socket=/var/run/libvirt/virtqemud-sock"
```

Comment:
It looks pretty weird that the socket for the session daemon would live in `/var/run`. Do you know the reason for that choice? My expectation would be that it either lives under `/run`, same as the system daemon, or under a more obviously unprivileged path.

Of course this is all pre-existing stuff, so no need to do anything about it in this PR :)

Reply:
Reading documentation it should live in a separate path: https://libvirt.org/manpages/virtqemud.html#when-run-as-non-root
I don't know the reasons behind the choice of `/var/run`, but maybe @xpivarc can. :slightly_smiling_face:

Reply:
That's because the session daemon usually runs under a regular user's account, which doesn't have permissions to create files under `/run`. In KubeVirt's case I think we're still running things as "pretend root" even in session mode, right? I'm really not sure how this interacts with containers, but the fact that the socket can be created under `/var/run` at all seems to confirm that it's not really entirely unprivileged.

Anyway, if you check out the virt-launcher you'll see that

```
$ ls -ld /var/run
lrwxrwxrwx. 1 root root 6 Jan  1  1970 /var/run -> ../run
```

so the distinction appears entirely pointless. Maybe I'm missing something, but I'd say let's just get rid of it and use `/run` unconditionally? It should really not change anything - except for being less confusing, that is ;)

Reply:
@andreabolognani 
>In KubeVirt's case I think we're still running things as "pretend root" even in session mode, right?

Yes, It was done to minimalize the differences in the dir structure.